.changeset/chatty-swans-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/adapter-node": minor
+---
+
+Updated the parsing of the BODY_SIZE_LIMIT environment variable from parseInt to parseFloat.
```

Comment:
```suggestion
breaking: allow any numeric value for `BODY_SIZE_LIMIT`, and interpret literally. Use `Infinity` rather than `0` for unrestricted body sizes
```